### PR TITLE
adds project_id to orders and estimates

### PR DESCRIPTION
**Pull request by thdaraujo**
### What
- adds project_id field to estimates and order creation
- adds metadata field to order creation
- adds changelog and updates to the readme
- updates the gem version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,24 +53,32 @@ In Patch, orders represent a purchase of carbon offsets or negative emissions by
 
 ```javascript
 // Create an order
-const mass = 1000000 // Pass in the mass in grams (i.e. 1 metric tonne)
-patch.orders.createOrder({ mass_g: mass })
+const mass = 1000000; // Pass in the mass in grams (i.e. 1 metric tonne)
+patch.orders.createOrder({ mass_g: mass });
+
+// You can also specify a project-id field (optional) to be used instead of the preferred one
+projectId = 'pro_test_1234'; // Pass in the project's ID
```
Comment (bspellacy) [inline README.md:60]: Can we add `const` here?

File: README.md [hunk 2]
```diff
@@ -53,24 +53,32 @@ In Patch, orders represent a purchase of carbon offsets or negative emissions by
...
+patch.orders.createOrder({ mass_g: mass, project_id: projectId });
+
+// Orders also accept a metadata field (optional)
+metadata = { user: 'john doe' };
```
Comment (bspellacy) [inline README.md:64]: Same comment for `const`

File: README.md [hunk 3]
```diff
@@ -53,24 +53,32 @@ In Patch, orders represent a purchase of carbon offsets or negative emissions by
...
+patch.orders.createOrder({ mass_g: mass, metadata: metadata });
 
 // Retrieve an order
-orderId = 'ord_test_1234' // Pass in the order's id
-patch.orders.retrieveOrder(orderId)
+orderId = 'ord_test_1234'; // Pass in the order's id
```
Comment (bspellacy) [inline README.md:68]: Same comment for `const`

File: README.md [hunk 4]
```diff
@@ -53,24 +53,32 @@ In Patch, orders represent a purchase of carbon offsets or negative emissions by
...
+patch.orders.retrieveOrder(orderId);
 
 // Place an order
-const orderId = 'ord_test_1234' // Pass in the order's id
-patch.orders.placeOrder(orderId)
+const orderId = 'ord_test_1234'; // Pass in the order's id
```
Comment (bspellacy) [inline README.md:72]: Could we remove the semi colons?

Comment (thdaraujo) [reply README.md]: I edited the linter configuration and it removed all the unneeded semicolons as well, not just the ones on this file.
Do you think it's better this way? I'm not a fan of semicolons as well.

https://prettier.io/docs/en/options.html#semicolons

**Pull request merged**